src/main/webapp/app/modules/canvas/visualization/visualization-modal/visualization-edit-modal/visualization-edit-modal.scss

```diff
@@ -0,0 +1,37 @@
+.properties-tab,
+.settings-tab {
+  overflow-y: auto;
+}
+._spectrum-TabsPanel-collapseWrapper_0248f {
+  position: sticky !important;
+  top: 0 !important;
+  z-index: 5 !important;
+  background-color: var(--spectrum-alias-background-color-default, var(--spectrum-global-color-gray-100));
+}
+.properties-tabs {
+  .field {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    position: relative;
+    top: 3px;
+  }
+  .fieldKey {
+    display: flex;
+  }
+  .fieldKey > div {
+    margin-left: 5px;
+  }
+}
+.featureName {
+  ._spectrum-Button-label_7a745 {
+    text-align: left;
+  }
+}
+.query {
+  color: var(--spectrum-alias-background-color-default, var(--spectrum-global-color-gray-100));
+  background-color: var(--spectrum-global-color-gray-800);
+}
```

Comment:
why are you overwrite the spectrum css? can be done using react spectrum props

Reply:
I am not overriding the spectrum css I just use spectrum css variable so its also work in dark theme